tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java

```diff
@@ -130,25 +134,31 @@ public void writeTo(ConsumerRecord<byte[], byte[]> consumerRecord, PrintStream o
                 } else {
                     output.print("NO_TIMESTAMP");
                 }
-                writeSeparator(output, printPartition || printOffset || printDelivery || printHeaders || printKey || printValue);
+                writeSeparator(output, anyTrue(printPartition, printOffset, printDelivery, printEpoch, printHeaders, printKey, printValue));
```

Comment:
Pardon me, what is the benefit of this helper?

Reply:
When I coding style like this: `a | b | c | d | e ... `, then I received a checkstyle ERROR: 
```
[ant:checkstyle] [ERROR] /Users/likangning/xijiu/kafka/tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java:137:32: Boolean expression complexity is 6 (max allowed is 5). [BooleanExpressionComplexity]
```

Reply:
How about adding `BooleanExpressionComplexity` to suppression?

Reply:
> How about adding `BooleanExpressionComplexity` to suppression?

Wonderful idea, I have fixed it, PTAL   @chia7712